### ISSUE TITLE
wrong token number

### ISSUE THREAD

**Issue by madswaord** [dscvr-one/ic-drip#2]
hello Dev, the ID Drip in my wallet shows #1102, but when i click the pics the link shows my is 1101, and the items was same as the 1101 on the raw data. please check.
<img width="1677" alt="截屏2021-09-10 下午6 28 01" src="https://user-images.githubusercontent.com/86275875/132840482-e554ca8e-727e-4da6-b127-07005da37ace.png">
![241631269846_ pic_hd](https://user-images.githubusercontent.com/86275875/132840574-424ab14f-5e81-4a97-b632-ebf3b729b1d0.jpg)
token id: qjtio-aqkor-uwiaa-aaaaa-ceadh-aaqca-aaarg-q
the link of the pics: https://d3ttm-qaaaa-aaaai-qam4a-cai.raw.ic0.app/?tokenId=1101

**Comment by rckprtr** [comment]
This looks like a display issue only on entrepot.  Looking into more.  Do you know your principal id for stoic wallet?

**Comment by madswaord** [comment]
> This looks like a display issue only on entrepot. Looking into more. Do you know your principal id for stoic wallet?

yes, it might be the issue of entrepot, the items and the # were same with datas in the file. check below for Pid, io25r-xzy5f-oicvh-5uhqz-r5wf5-oxik2-pfcl7-to5n7-cxaff-2iaac-eqe.

**Comment by bodily11** [comment]
Yes, this is a known Stoic Wallet error. This is being fixed right now. The image and Entrepot number is correct. The Stoic Wallet number will be updated soon.
